packages/desktop/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -228,7 +228,7 @@
                         bind:this={expirationTimePicker}
                         bind:value={expirationDate}
                         initialSelected={initialExpirationDate}
-                        disabled={disableChangeExpiration}
+                        disabled={disableChangeExpiration || isTransferring}
```

Comment:
Whats this change for?

Reply:
Probably some conflict resolving removed it, was included in this PR:

https://github.com/iotaledger/firefly/pull/6327

Reply:
Created the PR from that branch. Merging develop removed those.